shared/test/showcase/echo/operations_test.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "base64"
+require "test_helper"
+require "google/showcase/v1beta1/echo"
+require "grpc"
+
+class EchoTest < ShowcaseTest
+  def test_raise_invalid_get_operation
+    client = new_operations_client
```

Comment:
You can refactor all of the `client = new_operations_client` statements to a single `before` method if you want.

Reply:
Good idea - how's this?